acceptance/tests/direct_puppet/catalog_uuid_correlates_catalogs_with_reports.rb

```diff
@@ -1,4 +1,9 @@
 test_name "PUP-5872: catalog_uuid correlates catalogs with reports" do
+
+  tag 'audit:medium',
+      'audit:acceptance',
+      'server'
```

Comment:
seems like an agent test to me.  it doesn't need the server if we can give it a catalog fixture.